libflux/core/src/parser/mod.rs

```diff
@@ -1597,12 +1600,14 @@ impl Parser {
                     rbrack: self.make_comments(&end),
                 }))
             }
-            _ => {
+            t => {
                 let comma = self.expect(TokenType::Comma);
                 let mut items = vec![ArrayItem {
                     expression: init,
                     comma: self.make_comments(&comma),
                 }];
+                // keep track of the last token's byte offsets
+                let mut last = t.start_offset;
```

Comment:
Have to use `peek` here because the `expect` a few lines above can consume the token.

Reply:
Right, missed that one. One sec.